src/app/rooms/[roomId]/page.module.scss

```
@@ -408,16 +408,18 @@
 
   &__right {
     width: 100%;
-    max-height: 870px;
-    min-height: 0; // grid 자식이 overflow 처리하도록 설정
+    min-height: 0;
+    height: 870px;
     border-radius: 24px;
     border: 1.5px solid f.color(gray-100);
     box-shadow: 0 6px 12px 0 rgb(0 0 0 / 8%);
     transition:
```

Stylelint (17.2.0)

[error] Unexpected duplicate "transition" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)